src/lib.rs

```diff
@@ -100,6 +100,8 @@ extern crate lazy_static;
 extern crate log;
 #[macro_use]
 extern crate prometheus;
+#[macro_use]
+extern crate async_trait;
```

Comment:
Prefer to use `use` in where the macros are used rather than `#[macro_use] extern crate` because the macros are not used too often.